libs/langchain/langchain/graphs/neo4j_graph.py

```diff
@@ -89,17 +85,22 @@ def refresh_schema(self) -> None:
         """
         Refreshes the Neo4j graph schema information.
         """
-        node_properties = self.query(node_properties_query)
-        relationships_properties = self.query(rel_properties_query)
-        relationships = self.query(rel_query)
+        node_properties = [el["output"] for el in self.query(node_properties_query)]
+        rel_properties = [el["output"] for el in self.query(rel_properties_query)]
+        relationships = [el["output"] for el in self.query(rel_query)]
 
+        self.structured_schema = {
+            "node_props": {el["labels"]: el["properties"] for el in node_properties},
+            "rel_props": {el["type"]: el["properties"] for el in rel_properties},
+            "relationships": relationships,
+        }
         self.schema = f"""
         Node properties are the following:
```

Comment:
where are the existing labels represented in the schema?
or are they only implicit in the relationships?